app/routes/users/components/UserProfile.js

```diff
@@ -131,12 +167,36 @@ export default class UserProfile extends Component<Props, UpcomingEventsProps> {
       changeGrade
     } = this.props;
 
-    const { abakusGroups = [], firstName, lastName } = user;
+    const {
+      pastMemberships = [],
+      abakusGroups = [],
+      firstName,
+      lastName,
+      memberships = []
+    } = user;
 
-    const { groupsAsBadges = [], groupsAsPills = [] } = groupBy(
-      abakusGroups.filter(Boolean),
-      group => (group.logo ? 'groupsAsBadges' : 'groupsAsPills')
+    const { membershipsAsBadges = [], membershipsAsPills = [] } = groupBy(
+      memberships.filter(Boolean).map(membership => ({
+        ...membership,
+        abakusGroup: abakusGroups.find(g => g.id === membership.abakusGroup)
+      })),
+      membership =>
+        membership.abakusGroup.logo
+          ? 'membershipsAsBadges'
+          : 'membershipsAsPills'
     );
+
+    const {
+      pastMembershipsAsBadges = []
+      // pastMembershipsAsPills = []
```

Comment:
Comment?

Reply:
Just to make it easy for others to find the code, and understand that `pastMembershipsAsPills` is a thing.

Reply:
Guess we can remove it. Looks kinda strange